src/home/homectl.c

```diff
@@ -2664,12 +2706,20 @@ static int verb_firstboot(int argc, char *argv[], void *userdata) {
         if (r > 0) /* Already created users from credentials */
                 return 0;
 
-        r = has_regular_user();
-        if (r < 0)
-                return r;
-        if (r > 0) {
-                log_info("Regular user already present in user database, skipping user creation.");
+        r = getenv_bool("SYSTEMD_HOME_FIRSTBOOT_FORCE");
+        if (r == 0)
                 return 0;
```

Comment:
Hmm, _FORCE=0 doesn't necessarily imply that the whole user creation/detection logic shall be turned off? This behavior feels surprising for me. Either rename this to _OVERRIDE or perform has_regular_user() if r != 0?